integrations/ballerina/shopify-slack-order-notfication-service/README.md

```diff
@@ -0,0 +1,81 @@
+# Shopify Slack Order Notification Service
+
+Integrating Shopify with Slack and email notifications to streamline order processes and enhance the overall customer experience.
+
+## Prerequisites
+
+1. Shopify Partner Account. (You can create one [here](https://www.shopify.com/partners)).
+2. [Slack](https://slack.com/) subscription.
+3. [Ballerina](https://ballerina.io/) (Version 2201.7.1 is preferred).
+
+## Obtaing Configuration Values
+Need to obtain below secret values form your Shopify store and Slack.
+
+1. **Admin API access token from Shopify store app:** Create an app in your Shopify store with `read-order` permission and save the  **Admin API access token**.
+2. **API secret key from Shopify store app:** Obtain an **API secret key** from the same app.
+3. **Bot OAuth token form Slack app:** Create a Slack app via [Slack API](https://api.slack.com/) with `channels:write`, `channels:read`, `chat:write`, and `chat:write.public` scopes and obtained the **Bot OAuth token**.
+
+A slack channel named **orders** should be created.
+
+## Deploying in Choreo
+
+1. **Fork the Repository:** Fork the [choreo-examples repository](https://github.com/wso2/choreo-examples.git).
+
+2. **Create Event-triggered integration Component in Choreo:** 
+- Log into Choreo, navigate to **Create a New Component** page in your Choreo Project and select **Event-Triggered Integration**.
+- Fill out general details; name and description.
```

Comment:
do we need `;`?

Reply:
Fixed in - 3b91eb4dcf51bb35b601e40564c9a01c623b52c4